lib/wibox/init.lua

```diff
@@ -268,6 +270,18 @@ function wibox:find_widgets(x, y)
     return self._drawable:find_widgets(x, y)
 end
 
+function wibox:get_screen()
+    return capi.screen[ascreen.getbycoord(self.x, self.y)]
+end
+
+function wibox:set_screen(s)
+    s = capi.screen[s or 1]
+    if s ~= self:get_screen() then
+        self.x = s.geometry.x + 1
+        self.y = s.geometry.y + 1
```

Comment:
Assuming that this is the "+1" that you refer to in your commit: I think you can get rid of this again and have it fixed by #909.